### ISSUE TITLE
Build issue

### ISSUE THREAD

**Issue by mo5245** [febiz/h264-analysis-tool#1]
Do you have any of the build environment info and compiler used, getting make errors with gcc++ @ 98%

**Comment by heydarshahi** [comment]
I'm getting the same error! Were you able to solve it? 

CMake works fine. My problem starts when executing "make":

**> make[2]: *** No rule to make target '[/path/to/opencv/build]/lib/libopencv_cudabgsegm.so.3.3.1', needed by '../bin/H264AnalysisTool'.  Stop.**

I looked at the [/project/root/]build/CMakeFiles/H264AnalysisTool.dir/build.make file. In line 19, it declares:

> \# External object files for target H264AnalysisTool
> H264AnalysisTool_EXTERNAL_OBJECTS = ../bin/H264AnalysisTool: CMakeFiles/H264AnalysisTool.dir/src/main.cpp.o
> ../bin/H264AnalysisTool: CMakeFiles/H264AnalysisTool.dir/src/macroblockinfo.cpp.o
> ../bin/H264AnalysisTool: CMakeFiles/H264AnalysisTool.dir/src/vis.cpp.o
> ../bin/H264AnalysisTool: CMakeFiles/H264AnalysisTool.dir/src/zlib_buffer.cpp.o
> ../bin/H264AnalysisTool: CMakeFiles/H264AnalysisTool.dir/build.make
> ../bin/H264AnalysisTool: JM/libJM.a
> ../bin/H264AnalysisTool: [path/to/opencv/build]/lib/libopencv_cudabgsegm.so.3.3.1
> ../bin/H264AnalysisTool: [path/to/opencv/build]/lib/libopencv_cudaobjdetect.so.3.3.1
> ../bin/H264AnalysisTool: [path/to/opencv/build]/lib/libopencv_cudastereo.so.3.3.1
> ...

It seems like make is not seeing the OpenCV files and I'm not sure why.

**Comment by mo5245** [comment]
Yeah, had to use windows build environment with visual studio to get a successful compile.

**Comment by mo5245** [comment]
Don't remember all the details getting cmake and OpenCV all working in windows with VS but if you just want the binary you can get it here https://1drv.ms/u/s!AoGpIcwKwEhSgRdrSR-y5zgGbIxS

other useful info

Files have to be in the h264 container format, I used FFmpeg

`ffmpeg -i Input_File.asf  -vcodec copy -an -bsf:v dump_extra Output_File.264`

HotKeys:
Left/Right arrows = move frame back/forward 
i = show/hide info block
p = show/hide macro blocks
f = show/hide motion
m = show/hide color
g = show/hide grid
q = quit (must be used)
v= alt view (may not work with your video file)

**Comment by heydarshahi** [comment]
@mo5245 
Thanks for the reply and the binaries!
